eng/scripts/Language-Settings.ps1

```diff
@@ -92,19 +92,25 @@ function Get-dotnet-PackageInfoFromPackageFile ($pkg, $workingDirectory)
 # Return list of nupkg artifacts
 function Get-dotnet-Package-Artifacts ($Location)
 {
-  return Get-ChildItem "${Location}" -Recurse | Where-Object -FilterScript {$_.Name.EndsWith(".nupkg") -and -not $_.Name.EndsWith(".symbols.nupkg")}
+  $pkgs = Get-ChildItem "${Location}" -Recurse | Where-Object -FilterScript {$_.Name.EndsWith(".nupkg") -and -not $_.Name.EndsWith(".symbols.nupkg")}
+  if ($pkgs.Count -gt 1)
+  {
+    Write-Host "$($Location) should contain only one (1) published package"
+    Write-Host "No of Packages $($pkgs.Count)"
+    exit(1)
+  }
+  return $pkgs
 }
 
 # Stage and Upload Docs to blob Storage
 function Publish-dotnet-GithubIODocs ($DocLocation, $PublicArtifactLocation)
 {
-  $PublishedPkgs = Get-dotnet-Package-Artifacts $DocLocation
-  $PublishedDocs = Get-ChildItem "$($DocLocation)" | Where-Object -FilterScript {$_.Name.EndsWith("docs.zip")}
+  $PublishedPkgs = Get-dotnet-Package-Artifacts "${DocLocation}"
```

Comment:
```suggestion
  $PublishedPkgs = Get-dotnet-Package-Artifacts $DocLocation
```

If it isn't necessary to put quotes around it you can just keep the variable. In the other case I suggested only because I thought we needed quotes, but it might not be needed there either.